addons/base_import/controllers.py

```diff
@@ -7,12 +8,15 @@ class ImportController(Controller):
     @route('/base_import/set_file')
     def set_file(self, req, file, import_id, jsonp='callback'):
         import_id = int(import_id)
+        if file.filename[-3:] == "ods":
```

Comment:
Wouldn't it be a better idea to switch based on the file's content_type? That's what is being used elsewhere.